platform-sdk/docs/proposals/001-merkle-state-api/001-merkle-state-api.md

```diff
@@ -0,0 +1,74 @@
+# Merkle State API
+
+## Summary
+
+Refactor the Merkle tree so that it can be fully represented by the State API objects. Create a module that will provide simple and easy-to-use API for the Merkle tree.
```

Comment:
The summary should include a statement explaining WHY this proposal is needed. Why does the tree need to be fully represented by the State API?

Reply:
I added "Why" section to the document